### PR TITLE
feat(client): show alias in cloud instance bundle list

### PR DESCRIPTION
**Pull request by dreamlandliu**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/base/view.py [hunk 1]
```diff
@@ -151,6 +151,7 @@ def _print_history(
             "runtime": BaseTermView.place_holder_for_empty(),
         }
         data = BaseTermView.get_history_data(history, fullname)
+        print(data)
```
Comment (tianweidut) [inline client/starwhale/base/view.py:154]: Remove the debug print line?

**Review by tianweidut — CHANGES_REQUESTED** [review]

File: client/starwhale/base/view.py (hunk 1 above)
Comment (dreamlandliu) [reply client/starwhale/base/view.py]: done

File: client/starwhale/base/view.py [hunk 2]
```diff
@@ -232,15 +232,19 @@ def list_data(
...
-                result.append(
-                    {
-                        "name": _name,
-                        "version": _version,
-                        "tags": _v.get("tags", []),
-                        "size": _v.get("size", 0),
-                        "created_at": _v.get("created_at"),
-                    }
-                )
+                _item = {
```
Comment (tianweidut) [inline client/starwhale/base/view.py:235]:
```python
_item = {
                    "name": _name,
                    "version": _version,
                    "tags": _v.get("tags"),
                    "alias": _v.get("alias"),
                    "size": _v.get("size", 0),
                    "created_at": _v.get("created_at"),
                }
                if "alias" not in _v:
                    _item.pop("alias")
                if "tags" not in _v:
                    _item.pop("tags")
```
-->
```python
_item = {
                    "name": _name,
                    "version": _version,
                    "size": _v.get("size", 0),
                    "created_at": _v.get("created_at"),
                }
                if "alias" in _v:
                    _item["alias"] = _v["alias"]
                if "tags" in _v:
                    _item["tags"] = _v["tags"]
```
?

Comment (tianweidut) [reply client/starwhale/base/view.py:235]: 327-339 lines do the same change?

Comment (dreamlandliu) [reply client/starwhale/base/view.py:235]: The purpose of this practice is to display the fields in the order given. Would you have a better way?

Comment (tianweidut) [reply client/starwhale/base/view.py:235]: python dict type is unordered.

**Pull request closed without merging**